launchable/test_runners/pytest.py

```diff
@@ -137,14 +132,36 @@ def _pytest_formatter(test_path):
 def record_tests(client, json_report, source_roots):
 
     def data_builder(case: TestCase):
+        props = case.child(Properties)
+        result = {}
+        if props is not None:
+            """
+                Here is an example of an XML file with markers.
+                ```
+                    <properties>
+                        <property name="name" value="parametrize" />
+                        <property name="args" value="('y', [2, 3])" />
+                        <property name="kwargs" value="{}" />
+                        <property name="name" value="parametrize" />
+                        <property name="args" value="('x', [0, 1])" />
+                        <property name="kwargs" value="{}" />
+                    </properties>
+                ```
+            """
+            markers = []
+            marker: Dict[str, str] = {}
+            for prop in props:
+                markers.append({"name": prop.name, "value": prop.value})
```

Comment:
I guess you want to `marker.append(...)` not `markers`, right?

Reply:
You can write 

```python
if props is not None:
    markers = [{"name": prop.name, "value": prop.value} for prop in props]
    result["markers"] = markers if markers else []
```

Reply:
Nice suggestion. Let me fix it.